### PR TITLE
Add an intro about account/service/connection

### LINKED ISSUE #3463: [Feedback] Page: /getting-started/latest/services/
### Is it easy to find the information you need?

No

### Are the instructions clear?

No

### How could we improve the Timescale documentation site?

Hardly anything is explained here. Not sure What these functions are doing. Are they creating another table? If so, where? How does it relate to the table we created before this? So many questions.

### PR DESCRIPTION
**Pull request by atovpeko**

### CODE REVIEW

File: getting-started/services.md [hunk 1]
```diff
@@ -13,6 +13,14 @@ import ServiceOverview from "versionContent/_partials/_service-overview.mdx";
 
 # Create your first Timescale service
 
+To start using Timescale Cloud for your database, you need to take the following steps:
+
+1. Create a Timescale account: register in Timescale Console to get a centralized point for your database management.
+1. Create a Timescale service, that is, a PostgreSQL database instance, in Timescale Console.
```
Comment (billy-the-fish) [inline getting-started/services.md]:
```suggestion
1. Create a $SERVICE_LONG: a PostgreSQL database instance powered by [$TIMESCALE_DB](https://docs.timescale.com/#TimescaleDB), built for production, and extended with cloud features like transparent data tiering to object storage.
```

File: getting-started/services.md [hunk 2]
```diff
@@ -13,6 +13,14 @@ import ServiceOverview from "versionContent/_partials/_service-overview.mdx";
 
 # Create your first Timescale service
 
+To start using Timescale Cloud for your database, you need to take the following steps:
+
+1. Create a Timescale account: register in Timescale Console to get a centralized point for your database management.
```
Comment (billy-the-fish) [inline getting-started/services.md]:
```suggestion
1. Create a$COMPANY account: register in  $CONSOLE to get a centralized point to administer and interact with your data.
```

File: getting-started/services.md [hunk 3]
```diff
@@ -13,6 +13,14 @@ import ServiceOverview from "versionContent/_partials/_service-overview.mdx";
...
+1. Create a Timescale service, that is, a PostgreSQL database instance, in Timescale Console.
+1. Establish a connection with your newly created service to run queries on it and migrate your data from other sources, if necessary.
+
+This page takes you through those steps. It also explains how to create a standard PostgreSQL table in your service and convert it into a [hypertable][hypertables]. Anything you can do with regular PostgreSQL tables, you can do with hypertables, just with better performance and improved user experience for time-series data.
```
Comment (billy-the-fish) [inline getting-started/services.md]: Change this to a step

File: getting-started/services.md [hunk 4]
```diff
@@ -24,9 +32,7 @@ Timescale Cloud offers the following PostgreSQL optimized database services:
 
 <ServiceOverview />
 
-This section shows you how to create a service, connect to it, create a standard PostgreSQL table, then 
-convert it into a [Hypertable][hypertables]. Anything you can do with regular PostgreSQL tables, you can 
-do with hypertables, just with better performance and improved an user experience for time-series data.
+
```
Comment (billy-the-fish) [inline getting-started/services.md:32]: And add an intro sentence infront of every procedure to give a longer explanation of what it is all about.

**Review by billy-the-fish — CHANGES_REQUESTED** [review]

File: getting-started/services.md [hunk 5]
```diff
@@ -11,30 +11,34 @@ import Connect from "versionContent/_partials/_cloud-connect.mdx";
...
-# Create your first Timescale service
+# Create your first $SERVICE_LONG
 
-Timescale Cloud offers the following PostgreSQL optimized database services:
+$CLOUD_LONG offers the following PostgreSQL optimized database services:
 
-- **Time-series and Analytics**: a Timescale DB instance optimized for your 
+- **Time-series and Analytics**: a $TIMESCALE_DB instance optimized for your 
     [time-series and analytics][what-is-time-series] workloads.
 - **Dynamic PostgreSQL**: for all other workloads. A PostgreSQL instance with a
```
Comment (billy-the-fish) [inline getting-started/services.md:20]:
```suggestion
- **PostgreSQL**: for all other workloads. A PostgreSQL instance with a 
```

Please swap the order so it matches.

File: getting-started/services.md [hunk 6]
```diff
@@ -11,30 +11,34 @@ import Connect from "versionContent/_partials/_cloud-connect.mdx";
...
 
-This section shows you how to create a service, connect to it, create a standard PostgreSQL table, then 
-convert it into a [Hypertable][hypertables]. Anything you can do with regular PostgreSQL tables, you can 
-do with hypertables, just with better performance and improved an user experience for time-series data.
+To start using $CLOUD_LONG for your data:
+
+1. [Create a $COMPANY account][creatre-an-account]: register in $CONSOLE to get a centralized point to administer and interact with your data.
+1. [Create a $SERVICE_LONG][creatre-a-service]: that is, a PostgreSQL database instance, powered by [$TIMESCALE_DB][timescaledb], built for production, and extended with cloud features like transparent data tiering to object storage.
+1. [Connect to your $SERVICE_LONG][connect-to-your-service]: to run queries, add and migrate your data from other sources.
+1. [Create a hypertable][create-a-hypertable]: create a standard PostgreSQL table in your service, the convert it into a [hypertable][hypertables].
```
Comment (jamessewell) [inline getting-started/services.md]: the convert -> then convert

File: getting-started/services.md [hunk 7]
```diff
@@ -11,30 +11,34 @@ import Connect from "versionContent/_partials/_cloud-connect.mdx";
...
+- **Vector and AI**: for apps that require fast search on vector embeddings and metadata, like semantic search, image search, RAG and Agents. Powered by $COMPANY Vector and available on Time-series and Dynamic PostgreSQL services. 
 
 <ServiceOverview />
 
-This section shows you how to create a service, connect to it, create a standard PostgreSQL table, then 
-convert it into a [Hypertable][hypertables]. Anything you can do with regular PostgreSQL tables, you can 
-do with hypertables, just with better performance and improved an user experience for time-series data.
+To start using $CLOUD_LONG for your data:
+
+1. [Create a $COMPANY account][creatre-an-account]: register in $CONSOLE to get a centralized point to administer and interact with your data.
```
Comment (jamessewell) [inline getting-started/services.md]: Show creatre be create? I know it's not visible 😅

**Review by billy-the-fish — APPROVED** [review]

**Pull request merged**